File: practice/exercism/go/raindrops/raindrops.go
// Package raindrops ...
package raindrops

import "strconv"

// Convert converts a number to a certain string, depending on factors.
func Convert(n int) string {
	s := ""

	if n%3 == 0 {
		s += "Pling"
	}

	if n%5 == 0 {
		s += "Plang"
	}

	if n%7 == 0 {
		s += "Plong"
	}

	if len(s) > 0 {
		return s
	}

	return strconv.Itoa(n)
}

File: practice/exercism/kotlin/raindrops/src/main/kotlin/Raindrops.kt
object Raindrops {
    fun convert(num: Int): String {
        var str = if (num % 3 == 0) "Pling" else ""
        str += if (num % 5 == 0) "Plang" else ""
        str += if (num % 7 == 0) "Plong" else ""
        return if (str.length > 0) str else num.toString()
    }
}